src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2909,6 +2909,67 @@ curl -X GET \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### Get Product Limits for Plan
+
+To keep services stable for all clients, LiveChat limits common resources. The limits are different for each pricing plan.
+Method `check_products_limits_for_plan` shows what resources exceed its limits for any given plan. This information is useful before downgrading the plan.
+
+Limited resources in LiveChat:
+
+|                        |                                                                                                                     |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| `agents`               | The max number of active agents __*__                                                                               |
+| `groups`               | The max number of groups that can be created                                                                        |
+| `banned`               | The max number of banned customers or IPs                                                                           |
+| `queue`                | The max number of chats waiting in the queue ([Messaging Mode](/messaging#messaging-mode) is off)                   |
+| `queue_continuous`     | The max number of chats waiting in the queue ([Messaging Mode](/messaging#messaging-mode) is on)                    |
+| `groups_per_agent`     | The max number of groups an agent can be assigned to                                                                |
+| `access_rules`         | The max number of [Access Rules](/management/configuration-api#auto-access)                                         |
+| `group_chooser_groups` | The max number groups in [pre-chat survey](https://www.livechat.com/help/dividing-live-chat-by-group/#target-field) |
+
+__*__ Including bots, excluding suspended or awaiting approval agents
+
+#### Specifics
+
+|                 |                                                                                       |
+| --------------- | ------------------------------------------------------------------------------------- |
+| HTTP Method     | **GET**                                                                               |
+| Method URL      | `https://api.livechatinc.com/v3.5/configuration/action/check_product_limits_for_plan` |
+| Required scopes | `subscription.lc:rw`                                                                  |
```

Comment:
These 2 tables don't have headers, is that intentional?

Reply:
Fixed for the first table, the second one should be without a header.